docs/site/content/docs/tanzu-diagnostics.md

```diff
@@ -16,7 +18,20 @@ Other requirements include:
 * Access to a management cluster and its managed workload clusters (if needed)
 * Access to a local unmanaged cluster
 
-## Collecting diagnostics
+### Before you Begin
+
+If you want to debug the bootstrap cluster, first, determine the name of the bootstrap cluster:
+
+```sh
+kind get clusters
```

Comment:
We don't require the user to install `kind` as part of a normal install. I wonder if we should do something like `kubectl config get-contexts` or `docker ps` to find the cluster.

Reply:
This was added for #https://github.com/vmware-tanzu/community-edition/issues/1716
I think finding the kind cluster name here is in the context of a failed management cluster deploy where the kind cluster was installed but the cluster failed to deploy and delete the kind cluster

The 2nd paragraph in the introduction to this topic explains this - and this content originally came from a video from @joshrosso :

>When debugging a failed management cluster deployment, the kind bootstrap cluster is the key to being able to introspect and understand what is happening. Any issues or errors that occur in the kind bootstrap cluster during bootstrapping will provide information about potential problems in the final management cluster on the target provider. If the bootstrap process is stuck or did not finish successfully, use the diagnostics plugin to collect logs and cluster information.

Reply:
1716 didn't add requiring `kind`. It looks like that added a way to look up the name without needing to resort to `kind`. I think we should stick with that so we are not telling end users to install yet another thing, just to look up a name.

Reply:
Will the context be automatically set to the kind cluster during bootstrapping?

If so, we can simplify it even further by suggesting the user inspect the current context with `kubectl config view --minify`. if not, they will have to look at all the contexts and grab the name from there.

---

+1 - I agree with Sean, i don't think the `kind` CLI is a requirement here and the same can be achieved easily with an existing pre-req, `kubeclt`.

Reply:
OK thanks, I worked in both of those commands now